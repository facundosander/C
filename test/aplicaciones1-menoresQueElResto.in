crearListaJugadores

# creo una lista con personas y las siguientes edades: 22, 27, 25, 30, 45, 35

crearJugada
1 1 2 3 4
agregarEnPartida
crearJugador 1 35 Uruguay Juan
crearFecha 1/12/2022
insertarJugadorListaJugadores

crearJugada
3 3 2 2 1
agregarEnPartida
crearJugador 5 45 Brasil Roberto
crearFecha 5/12/2022
insertarJugadorListaJugadores

crearJugada
3 1 2 3 4
agregarEnPartida
crearJugador 3 30 Uruguay Luis
crearFecha 1/1/2023
insertarJugadorListaJugadores

crearJugada
3 3 3 4 4
agregarEnPartida
crearJugador 4 25 Uruguay Andres
crearFecha 8/1/2023
insertarJugadorListaJugadores

crearJugada
3 2 4 2 4
agregarEnPartida
crearJugador 2 27 Uruguay Rodrigo
crearFecha 10/1/2023
insertarJugadorListaJugadores

crearJugada
3 2 3 3 4
agregarEnPartida
crearJugador 7 22 Uruguay Matias
crearFecha 2/12/2023
insertarJugadorListaJugadores

# Imprimimos de Mayor a Menor
imprimirMayorAMenorTJugadoresLDE


# la pila de menores es la siguiente: 35, 30, 25, 22
menoresQueElResto

liberarListaJugadores

crearListaJugadores

# creo una lista con personas y las siguientes edades: 10, 8, 4, 3, 1

crearJugada
1 1 2 3 4
agregarEnPartida
crearJugador 1 1 Uruguay Emilia
crearFecha 1/12/2022
insertarJugadorListaJugadores

crearJugada
3 3 2 2 1
agregarEnPartida
crearJugador 5 3 Brasil Ines
crearFecha 5/12/2022
insertarJugadorListaJugadores

crearJugada
3 1 2 3 4
agregarEnPartida
crearJugador 3 4 Uruguay Augusto
crearFecha 1/1/2023
insertarJugadorListaJugadores

crearJugada
3 3 3 4 4
agregarEnPartida
crearJugador 4 8 Uruguay Pedro
crearFecha 2/2/2023
insertarJugadorListaJugadores

crearJugada
3 2 4 2 4
agregarEnPartida
crearJugador 2 10 Uruguay Noel
crearFecha 3/3/2023
insertarJugadorListaJugadores

# Imprimimos de Mayor a Menor
imprimirMayorAMenorTJugadoresLDE

# la pila de menores es la siguiente: 1
menoresQueElResto

liberarListaJugadores

crearListaJugadores

# creo una lista con personas y las siguientes edades: 1, 3, 8, 10, 20

crearJugada
1 1 2 3 4
agregarEnPartida
crearJugador 1 20 Uruguay Maxi
crearFecha 1/12/2022
insertarJugadorListaJugadores

crearJugada
3 3 2 2 1
agregarEnPartida
crearJugador 5 10 Brasil Augustina
crearFecha 5/12/2022
insertarJugadorListaJugadores

crearJugada
3 1 2 3 4
agregarEnPartida
crearJugador 3 8 Uruguay Lucia
crearFecha 1/1/2023
insertarJugadorListaJugadores

crearJugada
3 3 3 4 4
agregarEnPartida
crearJugador 4 3 Uruguay Sofia
crearFecha 2/2/2023
insertarJugadorListaJugadores

crearJugada
3 2 4 2 4
agregarEnPartida
crearJugador 2 1 Uruguay Alicia
crearFecha 3/2/2023
insertarJugadorListaJugadores

# Imprimimos de Mayor a Menor
imprimirMayorAMenorTJugadoresLDE

# la pila de menores es la siguiente: 20, 10, 8, 3, 1
menoresQueElResto

liberarListaJugadores

Fin