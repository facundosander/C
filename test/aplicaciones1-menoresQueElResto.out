1>2>#  creo una lista con personas y las siguientes edades: 22, 27, 25, 30, 45, 35.
3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>18>19>20>21>22>23>24>25>26>27>28>29>30>31>32>33>#  Imprimimos de Mayor a Menor.
34>Jugador 7:  Matias, Uruguay, 22 años
Jugada: 3 
Movimientos: Izquierda Arriba Arriba Abajo 
2/12/2023
Jugador 2:  Rodrigo, Uruguay, 27 años
Jugada: 3 
Movimientos: Izquierda Abajo Izquierda Abajo 
10/1/2023
Jugador 4:  Andres, Uruguay, 25 años
Jugada: 3 
Movimientos: Arriba Arriba Abajo Abajo 
8/1/2023
Jugador 3:  Luis, Uruguay, 30 años
Jugada: 3 
Movimientos: Derecha Izquierda Arriba Abajo 
1/1/2023
Jugador 5:  Roberto, Brasil, 45 años
Jugada: 3 
Movimientos: Arriba Izquierda Izquierda Derecha 
5/12/2022
Jugador 1:  Juan, Uruguay, 35 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
1/12/2022
35>#  la pila de menores es la siguiente: 35, 30, 25, 22.
36>Jugador 1:  Juan, Uruguay, 35 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
Jugador 3:  Luis, Uruguay, 30 años
Jugada: 3 
Movimientos: Derecha Izquierda Arriba Abajo 
Jugador 4:  Andres, Uruguay, 25 años
Jugada: 3 
Movimientos: Arriba Arriba Abajo Abajo 
Jugador 7:  Matias, Uruguay, 22 años
Jugada: 3 
Movimientos: Izquierda Arriba Arriba Abajo 
37>38>39>#  creo una lista con personas y las siguientes edades: 10, 8, 4, 3, 1.
40>41>42>43>44>45>46>47>48>49>50>51>52>53>54>55>56>57>58>59>60>61>62>63>64>65>#  Imprimimos de Mayor a Menor.
66>Jugador 2:  Noel, Uruguay, 10 años
Jugada: 3 
Movimientos: Izquierda Abajo Izquierda Abajo 
3/3/2023
Jugador 4:  Pedro, Uruguay, 8 años
Jugada: 3 
Movimientos: Arriba Arriba Abajo Abajo 
2/2/2023
Jugador 3:  Augusto, Uruguay, 4 años
Jugada: 3 
Movimientos: Derecha Izquierda Arriba Abajo 
1/1/2023
Jugador 5:  Ines, Brasil, 3 años
Jugada: 3 
Movimientos: Arriba Izquierda Izquierda Derecha 
5/12/2022
Jugador 1:  Emilia, Uruguay, 1 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
1/12/2022
67>#  la pila de menores es la siguiente: 1.
68>Jugador 1:  Emilia, Uruguay, 1 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
69>70>71>#  creo una lista con personas y las siguientes edades: 1, 3, 8, 10, 20.
72>73>74>75>76>77>78>79>80>81>82>83>84>85>86>87>88>89>90>91>92>93>94>95>96>97>#  Imprimimos de Mayor a Menor.
98>Jugador 2:  Alicia, Uruguay, 1 años
Jugada: 3 
Movimientos: Izquierda Abajo Izquierda Abajo 
3/2/2023
Jugador 4:  Sofia, Uruguay, 3 años
Jugada: 3 
Movimientos: Arriba Arriba Abajo Abajo 
2/2/2023
Jugador 3:  Lucia, Uruguay, 8 años
Jugada: 3 
Movimientos: Derecha Izquierda Arriba Abajo 
1/1/2023
Jugador 5:  Augustina, Brasil, 10 años
Jugada: 3 
Movimientos: Arriba Izquierda Izquierda Derecha 
5/12/2022
Jugador 1:  Maxi, Uruguay, 20 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
1/12/2022
99>#  la pila de menores es la siguiente: 20, 10, 8, 3, 1.
100>Jugador 1:  Maxi, Uruguay, 20 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
Jugador 5:  Augustina, Brasil, 10 años
Jugada: 3 
Movimientos: Arriba Izquierda Izquierda Derecha 
Jugador 3:  Lucia, Uruguay, 8 años
Jugada: 3 
Movimientos: Derecha Izquierda Arriba Abajo 
Jugador 4:  Sofia, Uruguay, 3 años
Jugada: 3 
Movimientos: Arriba Arriba Abajo Abajo 
Jugador 2:  Alicia, Uruguay, 1 años
Jugada: 3 
Movimientos: Izquierda Abajo Izquierda Abajo 
101>102>Fin.
